src/main/java/org/fundacion/jala/converter/view/converter/ImageConverterInterface.java

```diff
@@ -0,0 +1,154 @@
+    private final int FONT_SIZE = 12;
+    private final int MARGIN_SPACE = 30;
+    private final int MARGIN_BOTTOM_MAIN_CONTAINER = 200;
+    private final int MARGIN_BOTTOM_BTN_CONTAINER = 100;
+    private final int BTN_TYPE = 2;
+    private SelectFile file;
+    private ConvertTypeSelectImage imageSelect;
+    private OutputSettingsImage settings;
+    private ClientRequest clientRequest = new ClientRequest();
+    private String token;
+    private String checksumLocal;
+    private JLabel label;
+
+    public ImageConverterInterface(final String newToken) {
+        this.token = newToken;
+        JLabelStyle imageTitle = new JLabelStyle("Image converter", "h2");
+        imageTitle.setAlignmentX(LEFT_ALIGNMENT);
+        BtnStyle convertImage = new BtnStyle("Convert", BTN_TYPE);
+        convertImage.setAlignmentX(LEFT_ALIGNMENT);
+        convertImage.setFont(new Font("Barlow", Font.PLAIN, FONT_SIZE));
+        convertImage.addActionListener(this::actionPerformed);
+        file = new SelectFile();
+        file.setAlignmentX(LEFT_ALIGNMENT);
+        imageSelect = new ConvertTypeSelectImage();
+        imageSelect.setAlignmentX(LEFT_ALIGNMENT);
+        setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        setBorder(new EmptyBorder(TOP_BORDER, LEFT_BORDER, BOTTOM_BORDER, RIGHT_BORDER));
+        settings = new OutputSettingsImage();
+        settings.setAlignmentX(LEFT_ALIGNMENT);
+        ImageIcon icon = new ImageIcon("img/loading.gif");
+        label = new JLabel();
+        label.setIcon(new ImageIcon(icon.getImage()));
+        label.setVisible(false);
+        JPanel btnContainer = new JPanel();
+        btnContainer.setLayout(new FlowLayout(FlowLayout.CENTER));
+        btnContainer.add(convertImage);
+        JPanel container = new JPanel();
+        container.setBorder(new EmptyBorder(MARGIN_SPACE, 0, MARGIN_BOTTOM_BTN_CONTAINER, 0));
+        container.setLayout(new BoxLayout(container, BoxLayout.Y_AXIS));
+        container.add(file);
+        container.add(imageSelect);
+        container.add(settings);
+        setLayout(new BorderLayout());
+        setBorder(new EmptyBorder(0, MARGIN_SPACE, MARGIN_BOTTOM_MAIN_CONTAINER, MARGIN_SPACE));
+        add(imageTitle, BorderLayout.NORTH);
+        add(container, BorderLayout.CENTER);
+        add(btnContainer, BorderLayout.SOUTH);
+    }
+
+    /**
+     * Invoked when an action occurs.
+     *
+     * @param e the event to be processed
+     */
+    @Override
+    public void actionPerformed(final ActionEvent e) {
+        LOGGER.info("start");
+        if (checkFileSelect(file.getOriginFilePath())) {
+            label.setVisible(true);
+            try {
+                LOGGER.info("Execute Try");
+                checksumLocal = getFileChecksum(file.getOriginFilePath());
+                int option = JOptionPane.showConfirmDialog(this, "File Path: "
+                        + file.getOriginFilePath() + "\nConvert to: " + imageSelect.getConvertTo()
```

Comment:
Line code accept 120 characters, maybe some of this lines can be move up

Reply:
Thank you Pao, I'll move it